company-profiles/rainforest-qa.md

```diff
@@ -2,10 +2,50 @@
 
 ## Company blurb
 
-⚠ We don't have much information about this company yet!
+Rainforest QA is an on-demand QA solution. It’s our mission to enable
+development teams to deliver bug-free software while moving at the speed of
+continuous delivery. We are truly a global team, allowing us to bring together
+the best and most diverse talent. Our commitment to the distributed team model
+and to our company values has earned us multiple culture and workplace awards
+and helped us build a diverse team of individuals working toward the same goal:
+change the way QA is done.
```

Comment:
The formatting of this paragraph is odd - was it copy and pasted from somewhere? The line breaks don't naturally flow.

Reply:
This actually works fine on both Netlify and GitHub:

- https://deploy-preview-719--remote-jobs.netlify.com/rainforest-qa/
- https://github.com/remoteintech/remote-jobs/blob/5c347fbe243e22dd0d5352973f8afbcf2373ef10/company-profiles/rainforest-qa.md

So I'm going to go ahead and merge this, the formatting can also be cleaned up in a separate PR if you want.